schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+	id 	INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY NOT NULL,
+	name VARCHAR(25) NOT NULL,
+	date_of_birth DATE,
+	escape_attempts INT,
+	neutered BOOL,
```

Comment:
- [ ] It would be better if we replace `BOOL` with `boolean` instead for the data type